shared-libs/lineage/test/integration.js

```diff
@@ -808,22 +827,223 @@ describe('Lineage', function() {
         });
     });
 
-    it('processing a doc with itself as a parent errors out', function() {
+    it('processing a doc with itself as a parent does not error out', function() {
```

Comment:
Apparently, all of these tests were failing silently. 
https://travis-ci.org/github/medic/cht-core/jobs/677975167#L5892

Reply:
OUCH! I've raised an issue to try and detect this in future: https://github.com/medic/cht-core/issues/6369